data/resources/style.css

```diff
@@ -196,32 +196,30 @@ messagebubble.document .file > overlay {
   min-height: 40px;
 }
 
-messagebubble.document .file > overlay > image {
-  background: @accent_color;
-  color: @window_bg_color;
-  transition: opacity 200ms cubic-bezier(0.25, 0.46, 0.45, 0.94);
+messagebubble.document .file > overlay > statusindicator > image {
   padding: 12px;
+}
+
+messagebubble.document .file > overlay > statusindicator {
+  color: @accent_color;
+  transition: opacity 200ms cubic-bezier(0.25, 0.46, 0.45, 0.94);
   border-radius: 9999px;
 }
 
-messagebubble.document.outgoing .file > overlay > image {
-  background: @accent_fg_color;
-  color: @accent_bg_color;
+messagebubble.document.outgoing .file > overlay > statusindicator {
+  /* depends on bubble color */
+  color: #79c271;
```

Comment:
Mh, I think it makes more sense to use the accent color here, since we want it to be configurable in the future. Here we're just a random color.

Reply:
I would like to have that color as a separate because it actually depends on the bubble color. For example on my phone I use dark bubble colors even in dark mode and that color is white for them

Reply:
I can move it into a variable. Something like `@outgoing_message_fg_secondary`

Reply:
![image](https://github.com/paper-plane-developers/paper-plane/assets/63008755/cfe495a5-da2b-448e-80b8-a807cbef1e1c)

Looks like the required color is already defined in the theme